include/openthread/coap_secure.h

```diff
@@ -67,14 +67,28 @@ extern "C" {
 
 #define OT_DEFAULT_COAP_SECURE_PORT 5684 ///< Default CoAP Secure port, as specified in RFC 7252
 
+/**
+ * Coap secure connection event types.
+ *
+ */
+typedef enum otCoapSecureConnectEvent
+{
+    OT_COAP_SECURE_CONNECTED = 0,             ///< Connection established
+    OT_COAP_SECURE_DISCONNECTED_PEER_CLOSED,  ///< Disconnected by peer
+    OT_COAP_SECURE_DISCONNECTED_LOCAL_CLOSED, ///< Disconnected locally
+    OT_COAP_SECURE_DISCONNECTED_MAX_ATTEMPTS, ///< Disconnected due to reaching the maximum number of connection
+                                              ///< attempts.
```

Comment:
Same here, if we can fit in the same line:
```suggestion
    OT_COAP_SECURE_DISCONNECTED_MAX_ATTEMPTS, ///< Disconnected due to reaching the max conn attempts.
```

Reply:
Done